src/parser/rogue/shared/resources/EnergyCapTracker.js

```diff
@@ -54,15 +54,19 @@ class EnergyCapTracker extends RegenResourceCapTracker {
     return Math.floor(max);
   }
 
+  get wastedPercent() {
+    return this.missedRegen / this.naturalRegen || 0;
```

Comment:
I'm not sure by the operator precedency, I'd put parentheses to be sure
```suggestion
    return (this.missedRegen / this.naturalRegen) || 0;
```